dashboard/app/controllers/pd/professional_learning_landing_controller.rb

```diff
@@ -5,10 +5,6 @@ class Pd::ProfessionalLearningLandingController < ApplicationController
 
   def index
     view_options(full_width: true, responsive_content: true, no_padding_container: true)
-    if Pd::Enrollment.for_user(current_user).empty? && Plc::UserCourseEnrollment.where(user: current_user).empty?
-      redirect_to CDO.code_org_url('educate/professional-learning', CDO.default_scheme)
-      return
-    end
```

Comment:
This is the code that removes the conditional redirect to https://code.org/educate/professional-learning.